src/java/growthcraft/bees/init/GrcBeesRecipes.java

```diff
@@ -43,6 +43,8 @@
 import net.minecraftforge.fluids.Fluid;
 import net.minecraftforge.fluids.FluidContainerRegistry;
 import net.minecraftforge.fluids.FluidStack;
+import growthcraft.api.core.GrcFluid;
```

Comment:
Move these up to the growthcraft imports, also added the FluidRegistry import here